deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: Auto Pilot
     categories: "Storage"
     description: Cloud native storage solution for production workloads
-    containerImage: openstorage/openstorage-operator:master
+    containerImage: portworx/px-operator:1.5.0-dev
```

Comment:
here we are not using master tag?

Reply:
This is the 1.5.0 CSV. Here we would be using 1.5.0-dev.
The master tag will be used in the 2.0.0 CSV that is added in this PR - https://github.com/libopenstorage/operator/pull/289/files#diff-0b2a9ce703ba4233fbacb6e85b3d8ddb3c927e87365a57427bcf8088d53c960aR10